projects/schematics/src/migrations/mechanism/rename-symbol/rename-symbol_spec.ts

```diff
@@ -0,0 +1,116 @@
+/// <reference types="jest" />
+
+import { getSystemPath } from '@angular-devkit/core';
+import { TempScopedNodeJsSyncHost } from '@angular-devkit/core/node/testing';
+import { HostTree, Tree } from '@angular-devkit/schematics';
+import {
+  SchematicTestRunner,
+  UnitTestTree,
+} from '@angular-devkit/schematics/testing';
+import * as shx from 'shelljs';
+import { runMigration, writeFile } from '../../../shared/utils/test-utils';
+
+const MIGRATION_SCRIPT_NAME = 'migration-v4-rename-symbol-02';
+
+const fileWithSimpleImport = `import { OtherComponent1, Test1Component } from "@spartacus/storefront";
+import { Test2Component } from "@spartacus/core";
+
+const array = [OtherComponent1, Test1Component, Test2Component];`;
+
+// -----------------------------------------------------------------------
+
+const fileWithSimpleImportWithAlias = `import { OtherComponent2 as Test, Test1Component } from "@spartacus/storefront";
+import { Test2Component } from "@spartacus/core";
+
+const array = [Test, Test1Component, Test2Component];`;
+
+// -----------------------------------------------------------------------
+
+const fileWithSimpleImportAndRename = `import { OtherComponent3, Test1Component } from "@spartacus/storefront";
+import { Test2Component } from "@spartacus/core";
+
+const array = [OtherComponent3, Test1Component, Test2Component];`;
+// -----------------------------------------------------------------------
+
+fdescribe('renamed symbols', () => {
+  let host: TempScopedNodeJsSyncHost;
+  let appTree = Tree.empty() as UnitTestTree;
```

Comment:
It's better if follow Angular here, and do `let appTree = new UnitTestTree(new EmptyTree());`.
Source: https://github.com/angular/angular-cli/blob/master/packages/schematics/angular/migrations/update-10/remove-es5-browser-support_spec.ts#L74

Reply:
Unfortunately once I tried to change it, lots of errors appear during unit tests. I check other schematics spec files and all of them follow the same approach which also I used. I will suggest leaving it in my PR like this and change it globally in another ticket.